### ISSUE TITLE
[bug]: docker compose up errors with exec /opt/invokeai/docker-entrypoint.sh: no such file or directory

### ISSUE THREAD

**Issue by skittixch** [invoke-ai/InvokeAI#4976]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Windows

### GPU

cuda

### VRAM

11GB

### What version did you experience this issue on?

InvokeAI v3.4.0rc1

### What happened?

cd into the InvokeAI, copy and rename .env.sample to .env. docker compose up. It creates the image, then complains that /opt/invokeai/docker-entrypoint.sh doesn't exist.
In my case, I've tried many different root paths for invoke, all error out similarly.

### Screenshots

![image](https://github.com/invoke-ai/InvokeAI/assets/762694/cfd3a54a-a590-4a7c-adbe-0b3e09be5c06)

### Additional context

I'm using WSL2, and have confirmed that I'm using buildkit (I think)

I can't be the only person with this problem, so I imagine it may be user error, but I'm running out of ideas to fix this. Any help is appreciated.

### Contact Details

eric@ericbacus.com

**Comment by skittixch** [comment]
I think I've got this figured out. the file "docker-entrypoint.sh" is saved with CRLF "End-of-line sequence", rather than LF. This is enough to make the docker compose up file fail on my windows/docker desktop/WSL2 system.

![image](https://github.com/invoke-ai/InvokeAI/assets/762694/0b36867f-51d8-404b-b926-6c3d41ef89ef)

Changing CRLF to LF has fixed my issue.

**Comment by ebr** [comment]
@skittixch thank you for digging into this. Glad you found the solution locally; I opened a PR that should fix the issue for anyone who clones this repo going forward. Please test it out if you have a chance.

**Comment by skittixch** [comment]
Thanks! I wasn't sure how to confirm that I'd have uploaded the proper
file, so left this open until someone could reliably fix it.
Notably, I found that I needed to run docker compose up from WSL directly,
and not from the cmd prompt or powershell, because it wouldn't be able to
find the invokeai root folder on the host machine since it would need
something like C:\path\to\root, rather than /mnt/c/path/to/root. not sure
if that makes sense, but running the command in WSL was the other gotcha
that I figured out. Do with that what you will
